manifests/s/Schinagl/LinkShellextension/3.9.3.5/Schinagl.LinkShellextension.installer.yaml

```diff
@@ -0,0 +1,19 @@
+# Created using wingetcreate 0.4.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Schinagl.LinkShellextension
+PackageVersion: 3.9.3.5
+InstallerSwitches:
+  Silent: /s

```

Comment:
This needs to be /S but it would be better to just mark the installer type as nullsoft

Reply:
fixed, thanks!